pkg/config/defaults.go

```diff
@@ -247,6 +247,7 @@ func setDefaults(local bool) {
 
 	viper.SetDefault(DatabaseLocalPathKey, DefaultDatabaseLocalPath)
 	viper.SetDefault(DatabaseLocalPrefetchSizeKey, DefaultDatabaseLocalPrefetchSize)
+	viper.SetDefault("database.local.sync_writes", true)
```

Comment:
Related?

Reply:
Yes - config.go's GetKVParams as normalize this to true by treat it as a pointer.
I'll update the code so it will look like all other defaults.